orchestration/dags/data_gcp_dbt/models/intermediate/gcp/int_gcp__bigquery_cost.sql

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
est ce qu'on veut mettre cette table dans intermediate ? elle servirait à faire un dashboard de suivi des couts sur Metabase..

Reply:
+1 added to mrt_monitoring